charts/airflow/values.yaml

```diff
@@ -1360,6 +1360,10 @@ dags:
     ##
     syncTimeout: 120
 
+    ## the git submodule behavior to use for syncs: one of 'recursive', 'shallow', or 'off'
+    ##
+    submoduleStrategy: recursive
```

Comment:
I think we can just call the value `submodules`, for consistency with the [git-sync command-line and environment variable parameters](https://github.com/kubernetes/git-sync#manual).